src/arfs/turbo.ts

```diff
@@ -1,64 +1,50 @@
 import { DataItem } from 'arbundles';
-import { createAxiosInstance } from '../utils/axiosClient';
-import { AxiosInstance } from 'axios';
+import { Readable } from 'node:stream';
 
-interface TurboParams {
-	turboUrl: URL;
+import {TurboUnauthenticatedClient, TurboUploadDataItemResponse, TurboFactory } from '@ardrive/turbo-sdk';
+
+export interface TurboSettings {
+	turboUploadUrl: URL;
+	turboPaymentUrl: URL;
 	isDryRun: boolean;
-	axios?: AxiosInstance;
 }
 
 export interface TurboCachesResponse {
 	dataCaches?: string[];
 	fastFinalityIndexes?: string[];
 }
 
-export interface SendDataItemsResponse extends TurboCachesResponse {
-	id: string;
-	owner: string;
-}
-
+// Note: this class is a wrapper of the TurboSDk - it's helpful for things like dry run and other tests, but could be removed in the future
 export class Turbo {
-	public readonly turboUrl: URL;
 	private isDryRun: boolean;
-	private axios: AxiosInstance;
+	private turbo: TurboUnauthenticatedClient;
 
-	constructor({ turboUrl, isDryRun, axios = createAxiosInstance({}) }: TurboParams) {
-		this.turboUrl = turboUrl;
+	constructor({ turboUploadUrl, turboPaymentUrl, isDryRun = false }: TurboSettings) {
 		this.isDryRun = isDryRun;
-		this.axios = axios;
-	}
-
-	private get dataItemEndpoint(): string {
-		return `${this.turboUrl.href}v1/tx`;
+		this.turbo = TurboFactory.unauthenticated({
+			uploadServiceConfig: {
+				url: turboUploadUrl.toString()
+			},
+			paymentServiceConfig: {
+				url: turboPaymentUrl.toString()
+			}
```

Comment:
I had to make this change to get the pathing to line up. On the SDK side we add `/v1` but toString() on URL type gives `https://upload.ardrive.io/`, so we go to `https://upload.ardrive.io//v1` and get a 404. `origin` returns `https://upload.ardrive.io` and that works. A better route might be to use the URL type on the SDK side (if we like that type) or start making some custom url logic like trimming any trailing `/` on we're compatible with both inputs 

```suggestion
			uploadServiceConfig: {
				url: turboUploadUrl.origin
			},
			paymentServiceConfig: {
				url: turboPaymentUrl.origin
			}
```